### DISCUSSION TITLE
UiAvatar Field - add randomColor() or allow customization of url

### DISCUSSION THREAD

**Discussion by mrvnklm** [laravel/nova-issues#5402] [Q&A] [answered]
**Issue**
When using UiAvatar Field I think one of the most important features is random color, which there is no possiblity to use with the field provided by Nova.

**Possible solution**
Add `randomColor()` method to set `&random=true` in URL or even allow `extendURL($string)` to be able to extend the URL with custom params

**Comment by crynobone** [comment, marked as answer]
```php
UiAvatar::make()->backgroundColor('random'),
```

![image](https://user-images.githubusercontent.com/172966/225566113-36dab6b9-a363-4d08-b666-e5ba6d56daa6.png)

**Reply by mrvnklm** [reply]
Thank you! Don't know why I didn't think of that.
